cmd/svcat/class/create_cmd.go

```diff
@@ -73,12 +67,22 @@ func (c *createCmd) Run() error {
 		return err
 	}
 
-	class.Spec.ExternalName = c.name
+	newClass := &v1beta1.ClusterServiceClass{
```

Comment:
I'm confused about why you changed this part? Previously, it would copy the spec of the original class, and save it with a new name which was the desired behavior. With this change, the new class is missing key information from the original and it wouldn't be usable.

My vote is for going back to what you had and making sure through tests that it does what we need without unwanted side effects (if that was the concern that made you change it?)

Reply:
Just realized that this was in response to one of Jonathan's comments above. I replied there but I believe we can go back to what you had before (I spoke with him in person about this last week) and just make sure that we have enough tests to verify that it works as we hoped, without odd side-effects.